server/server/app.py

```diff
@@ -34,7 +34,8 @@ def app_factory() -> Tuple[Api, Flask]:
     api.add_resource(Images, '/images/<string:division>/<int:vol>')
     api.add_resource(Epigraphs, '/epigraphs')
     api.add_resource(WhyWeRead, '/whyweread')
-    api.add_resource(Glossary, '/glossary/<string:glossword>')
+    api.add_resource(Glossary, '/glossary')
+    api.add_resource(Adjacent, '/adjacent/<string:word>')
```

Comment:
`adjacent` not descriptive enough as an URL path variable.

PS. Shouldn't it be `/glossary/adjacent/<string:word>`? Is `adjacent` a subtype of `glossary`?

Reply:
No, they are two very different things. The glossary is actually a separate dictionary with specific glossary terms. Adjacent terms are drawn from the sum of all words in all dictionaries together, sorted (preferably in pali order but we're not there yet) and then taken 5 above and 5 below the search-word.